make.bat

```diff
@@ -39,14 +39,27 @@ if "%1" == "help" (
 	echo.  check
 	echo.  serve      to serve devguide on the localhost (8000)
 	goto end
-)
+:help0
 
 if "%1" == "clean" (
 	for /d %%i in (%BUILDDIR%\*) do rmdir /q /s %%i
 	del /q /s %BUILDDIR%\*
 	goto end
 )
 
+rem If we get this far, we're going to use the SPHINXBUILD command
```

Comment:
Unless we got a "clean" "check", "serve", "help", or "" we should get here, right?

Reply:
Yes. The venv bit is here rather than up top for that reason. Perhaps it was someone's logic to order the targets so in anticipation. Happy to assume so.